docker-compose.yml

```diff
@@ -869,6 +870,28 @@ services:
     volumes: *ubuntu-volumes
     command: *python-command
 
+  ubuntu-swift:
+    # Usage:
+    #   docker-compose build ubuntu-cpp
+    #   docker-compose build ubuntu-swift
+    #   docker-compose run --rm ubuntu-swift
+    # Parameters:
+    #   ARCH: amd64, arm64v8, ...
+    #   UBUNTU: 20.04
+    image: ${REPO}:${ARCH}-ubuntu-${UBUNTU}-swift
+    build:
+      context: .
+      dockerfile: ci/docker/linux-apt-swift.dockerfile
+      args:
+        base: ${REPO}:${ARCH}-ubuntu-${UBUNTU}-cpp
```

Comment:
Do we need to use `ubuntu-cpp` as base image?

Should we use a Swift's Docker image instead? https://hub.docker.com/_/swift/?tab=tags

Reply:
Good idea, will do.